docs/setup/collected-reported-metrics.md

```diff
@@ -0,0 +1,117 @@
+- `github_actions_job_success_count`
+- `github_actions_job_cancelled_count`
+- `github_actions_job_inprogress_count`
+- `github_actions_job_queued_count`
+- `github_actions_job_total_count`
+
+## Cost metric
+
+This is the last metric we collect, and it is one of the most important
+ones. It allows us to determine the cost of our CI runs.
+
+### The formula to calculate the cost over a period of time
+
+To calculate this metric, we use the following formula:
+
+```
+cost = duration (per second) / 60 * cost (per minute)
+```
+
+### How do we find the cost per minute?
+
+#### GitHub
+
+As for GitHub, it is quite simple. They provide us with a fixed value, and
+the price never varies. To give an example, for `ubuntu-latest`, we have a cost
+of 0.008$/min, that's it. Easy!
+
+#### Self-Hosted
+
+When it comes to the cost of self-hosted runners, it's a bit more complicated.
+
+Scality's self-hosted runners include Google Cloud Provider (GCP) and AWS.
+
+The cost can be found based on the machine type in the Management Console
+for AWS (when creating an EC2 instance) and on the
+[Google Cloud website](https://cloud.google.com/compute/vm-instance-pricing)
+for GCP.
+
+Unfortunately, these values are not accurate as they lack several elements
+such as bandwidth or storage. As for storage costs, they can be found in
+the same places where the machine type cost is available. However, it is
+not possible to determine the bandwidth cost directly.
+
+To overcome this, we had to devise a workaround. We didn't necessarily
+need an exact cost for CI but rather a value close to reality (+/- 5%)
+for data visualization purposes.
+
+We analyzed previous invoices and calculated the additional cost generated
+by bandwidth, which amounted to approximately 30% for each month.
+Consequently, we were able to approximate the cost using the following formula:
+
+```
+cost = (cost_per_flavor + cost_per_storage) * 130 / 100
+```
+
+_Good news, GCP and AWS costs are quite the same for the same flavors._
+
+### The different tags and their associated cost
+
+| Provider                           | Runner                      | Cost ($ per min)   |
+| ---------------------------------- | --------------------------- | ------------------ |
+| GitHub                             | `ubuntu-latest`             | 0.008              |
+| GitHub                             | `ubuntu-18.04`              | 0.008              |
+| GitHub                             | `ubuntu-20.04`              | 0.008              |
+| GitHub                             | `ubuntu-22.04`              | 0.008              |
+| GitHub                             | `ubuntu-20.04-4core`        | 0.016              |
+| GitHub                             | `ubuntu-22.04-4core`        | 0.016              |
+| GitHub                             | `ubuntu-22.04-8core`        | 0.032              |
+| AWS & GCP                          | `small`                     | 0.000625           |
```

Comment:
Add name of aws or gcp flavor, the large is the label we used internally

Reply:
I understand. Do you have a way to let the people at Scality know that the small, large, ... filters exist and can be used?

Reply:
Oh, I may have misunderstood. Do you want me to add the `machine_type` but keep the `flavors` column, or do you want me to remove the `flavors` column?